dev/AnimatedIcon/TestUI/AnimatedIconPage.xaml

```diff
@@ -0,0 +1,761 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
```

Comment:
There's a bunch of commented out markup in here. Intentional?

Reply:
Yeah, this highlights the change made to the checkbox template to integrate animated icon. I'll fix this when the checkbox change gets checked in.